### DISCUSSION TITLE
Difficulty to locate several cultivar-specific parameters for redefining while setting new maize cultivars

### DISCUSSION THREAD

**Discussion by ysharaby1102** [APSIMInitiative/ApsimX#9351] [Q&A] [answered]
Hello everyone,

I am currently attempting to incorporate new maize cultivars into APSIM next gen by adjusting hybrids and their specific parameters based on research papers such as "_Predicting crop yields and soil-plant nitrogen dynamics in the US Corn Belt_" and "_Calibration of a crop growth model in APSIM for 15 publicly available corn hybrids in North America_". However, I am having trouble locating some parameters in the APSIM GUI to redefine:

- leaf_app_rate1 (°C-d)
- leaf_app_rate2_3 (°C-d)
- nUptakeCease (°C-d)
- targetLeafSLN (g N/m2)
- partition_rate_leaf
- frac_stem2flower
- grainNfillrate (mg N grain−1°C-d−1)
- targetRachisNconc (%)
- targetStemNconc (%)
- rootdepthrate (mm/d)

I would greatly appreciate any help finding these parameters.

**Comment by BrianCollinss** [comment]
Right click on the Maize model in your simulation tree and make model structure visible. Then you can explore the model and find the parameters of interest. You can also find the path to those parameters by right clicking on them.

**Reply by ysharaby1102** [reply]
I know how to make the model visible and explore it, yet after a long exploration, I could not find these specific parameters...

**Reply by BrianCollinss** [reply]
Are you sure the list is for NextGen, and not APSIM Classic?

**Reply by ysharaby1102** [reply]
I am not sure if it is for APSIM Next-Gen, but this is part of my question - if there are correlative parameters in APSIM Next-Gen

**Reply by BrianCollinss** [reply]
I'm certain at least some of them exist in NextGen exactly the way they existed in Classic, but I'm also certain names are not necessarily kept. You need to compare the documents of the two version to be sure, or ask the developers of the Maize model about each parameter you're interested in.

**Reply by ysharaby1102** [reply]
@HamishBrownPFR @hut104 maybe you could help?

**Comment by hut104** [comment, marked as answer]
The following were in 7.10 with Next Generation equivalents in bold. Note, however, that the models are very different and the meaning of some values are slightly different and so exact values are different between 7.10 and Next Generation.  In some cases, there is no equivalent value and you will need to look for something similar to achieve the same purpose.

To do this, I've basically just gone looking through Classic to see how it described these, and then walked through the maize model looking for something with the same purpose.

    <leaf_app_rate1 type="text" description="leaf_app_rate1">65.0</leaf_app_rate1>
    <leaf_app_rate2 type="text" description="leaf_app_rate2">36.0</leaf_app_rate2>
    <leaf_app_rate3 type="text" description="leaf_app_rate3">36.0</leaf_app_rate3>

**.Simulations.Simulation.Field.Maize.Structure.Phyllochron.Phyllochron.Phyllochron.XYPairs**

    <nUptakeCease>523</nUptakeCease>
**No equivalent.  Look into the model and find an similar parameter required to achieve the 
same thing.**

    <targetLeafSLN>1.7</targetLeafSLN>

**No equivalent.  Look into the model and find an similar parameter required to achieve the 
same thing. e.g. critical or max N concs**

    <partition_rate_leaf description="coefficient of sigmoidal function between leaf partition fraction and internode no**2 (0-1)">0.0182</partition_rate_leaf>

**No equivalent.  Look into the model and find an similar parameter required to achieve the 
same thing. e.g. specific leaf areas**

    <frac_stem2flower description="fraction of dm allocated to stem that goes to developing head">0.17</frac_stem2flower>

** No equivalent.  Look into the model and find an similar parameter required to achieve the 
same thing. e.g. .Simulations.Simulation.Field.Maize.Stem.DMDemands.Structural.DMDemandFunction.PartitionFraction**

    <targetRachisNConc>0.01</targetRachisNConc>
**.Simulations.Simulation.Field.Maize.Rachis.MaximumNConc**

    <x_stem_n>3         5       7         10</x_stem_n>
    <targetStemNConc>0.04   0.04   0.01      0.005</targetStemNConc>
**.Simulations.Simulation.Field.Maize.Stem.MaximumNConc.XYPairs**

      stage_name    sow  ger  eme  juv    fi   flag    fl  st_gf end_gf   mat  hv_rpe  end_crop
      stage_code   = 1   2    3    4      5     6     7    8      9      10   11   12
      -->
    <root_depth_rate>0 33    33    33     33    33    33   0      0       0    0    0</root_depth_rate>
**.Simulations.Simulation.Field.Maize.Root.RootFrontVelocity.PotentialRootFrontVelocity**

**Reply by ysharaby1102** [reply]
@hut104 Thank you so much!
